util/patricia_trie/src/triedbmut.rs

```diff
@@ -160,14 +165,14 @@ enum Action<H: Hasher> {
 }
 
 // post-insert action. Same as action without delete
-enum InsertAction<H: Hasher> {
+enum InsertAction<H> {
```

Comment:
For my education, why is it better to have the type param be unbounded like this? Knowing that `H` is a `Hasher::Out` is – to my eyes at least – helpful to understand what is going on, so curious about the reasoning here.

Reply:
Ok, makes sense. I think rob pointed me to https://rust-lang-nursery.github.io/api-guidelines/future-proofing.html#c-struct-bounds at one time.